packages/appsync-modelgen-plugin/src/__tests__/visitors/__snapshots__/appsync-java-visitor.test.ts.snap

```diff
@@ -1,5 +1,25 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`AppSyncModelVisitor Many To Many V2 Tests Should generate the intermediate model successfully 1`] = `
```

Comment:
can we check the `PostTags` model that is generated?

Reply:
We're doing specific checks like this in the base appsync visitor tests by looking at the model map rather than pattern matching the code, it let me do a lot more specific and pointed checks